lib/pymedphys/docs/contrib/info/release-guide.md

```diff
@@ -0,0 +1,111 @@
+# pymedphys Release Procedure
+
+## Determine next release version
+
+pymedphys uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html) in the format `MAJOR.MINOR.PATCH`, the next release number should typically be `MAJOR.MINOR+1.PATCH`. For example, if the previous release was `0.38.0`, the upcoming release will be `0.39.0`.
+
+In instances where the only changes since the last release are bug fixes and none of the pymedphys API has changed, you should increment the `PATCH` value: `MAJOR.MINOR.PATCH+1`
+
+In the remainder of this guide this version code will be referred to as `VERSION`.
+
+## Create a branch to prepare the release
+
+Create a branch named `VERSION-release-prep`:
+
+```bash
+git checkout -b VERSION-release-prep
+git push --set-upstream origin 0.39.0-release-prep
+```
+
+## Update version in pyproject.toml
+
+Update the version code near the top of the file:
+
+```toml
+[tool.poetry]
+name = "pymedphys"
+version = "VERSION"
+readme = "README.rst"
+...
+```
+
+Then run poetry update as well as propagate:
+
+```bash
+poetry update
+poetry run pymedphys propagate
+```
+
+## Update CHANGELOG
+
+Amend the `CHANGELOG.md` file to describe the changes since the last release. Insert information for this release near the top of the file and populate the sections as appropriate (remove unused sections):
```

Comment:
Doesn't affect what you've written here, but I think it makes sense to enforce CHANGELOG edits at the time of contribution. We can add this is a future "how to make a contribution" page

Reply:
GitHub has a PR checklist option. @Matthew-Jennings, would you be open to setting that up and including a checkbox in the template for "added relevant changes to the changelog"?

Reply:
Yep, after around 10th October, I can get stuck into that. I'm going to be slammed between now and then.